src/tests/nativeaot/nativeaot.csproj

```diff
@@ -1,8 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
-    <ItemGroup>
-      <MergedWrapperProjectReference Include="*/**/*.??proj" Exclude="*/**/Library.csproj" />
-    </ItemGroup>
-  
-    <Import Project="$(TestSourceDir)MergedTestRunner.targets" />
-  </Project>
```

Comment:
It would be good to reduce the diff on this file by following the upstream (wrong) whitespace formatting.

Reply:
Done